src/commands/apphosting-rollouts-create.ts

```diff
@@ -7,13 +7,12 @@ import { createRollout } from "../apphosting/rollout";
 
 export const command = new Command("apphosting:rollouts:create <backendId>")
   .description("create a rollout using a build for an App Hosting backend")
-  .option("-l, --location <location>", "specify the region of the backend", "us-central1")
-  .option("-i, --id <rolloutId>", "id of the rollout (defaults to autogenerating a random id)", "")
+  .option("-l, --location <location>", "specify the region of the backend", "-")
```

Comment:
These flag changes are breaking changes, and ideally should be part of the next major version., unless App Hosting has a strong argument for allowing breaks in minor versions.